components/yaml/yaml_format.rst

```diff
@@ -300,4 +300,25 @@ Comments can be added in YAML by prefixing them with a hash mark (``#``):
     Comments are simply ignored by the YAML parser and do not need to be
     indented according to the current level of nesting in a collection.
 
+Explicity Typing
+----------------
+
+The YAML specification defines some tags to set the type of any data explicitly:
+
+.. code-block:: yaml
+
+    data:
+        # this value is parsed as a string (it's not transformed into a DateTime)
+        start_date: !str 2002-12-14
```

Comment:
I agree with @xabbuh. `!str` is "a Symfony thing" and it's wrong. Let's pretend it doesn't exist 😁